### PR TITLE
hack/munich22: Add blogpost for Munich hackathon

### PR DESCRIPTION
**Pull request by RaduNichita**
Signed-off-by: Radu Nichita <radunichita99@gmail.com>

### CODE REVIEW

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+But this one was special since it was organized in München.
+It was the 3rd international onsite hackathon, after the ones in Lyon and Aachen.
+From the Unikraft community, [Răzvan](https://github.com/razvand/) and [Radu](https://github.com/RaduNichita/) were there, whereas other community members provided support online, on Discord.
+
+Over `30` participants came to Technische Universität München for the 2-day event about unikernels. This is the highest attendance for an onsite Unikraft Hackathon onsite event!
+
+### Make open-source contributions great again!
+
+On the first day, there were sessions about configuring, building, running, and debugging applications with `Unikraft`.
+The participants seemed excited about the topics and dove deep into understanding the principles behind the unikernels. They were also very active in reporting some new issues.
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:20]:
```suggestion
The participants were given initial information about the topics and then they dove deep into understanding the principles behind the unikernels.
They were very active in reporting new issues.
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The second day started with Răzvan's spectacular presentation about `Binary Compatibility`.
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:22]:
```suggestion
The second day started with Răzvan's talk and demo on `Binary Compatibility`.
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Then, the folks got involved in solving hackathon challenges, from porting applications as `static-pie-apps` to fixing some existing issues).
+The results were quite impressive, with 8 PRs submitted during the day.
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:24]:
```suggestion
Results speak for themselves:

 - 6 PRs submitted as part of the [`static-pie-apps` repository](https://github.com/unikraft/static-pie-apps/pulls)
 - PR for [version bump of `newlib`](https://github.com/unikraft/lib-newlib/pull/27)
 - PR for (version bump of `zlib`](https://github.com/unikraft/lib-zlib/pull/4)
 - 2 issues in the [core `unikraft` repository](https://github.com/unikraft/unikraft/issues)
 
 All contributions were awarded points that made it to [the scoreboard](https://leaderboardhq.com/3wt61mcc).
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Over `30` participants came to Technische Universität München for the 2-day event about unikernels. This is the highest attendance for an onsite Unikraft Hackathon onsite event!
+
+### Make open-source contributions great again!
+
+On the first day, there were sessions about configuring, building, running, and debugging applications with `Unikraft`.
+The participants seemed excited about the topics and dove deep into understanding the principles behind the unikernels. They were also very active in reporting some new issues.
+
+The second day started with Răzvan's spectacular presentation about `Binary Compatibility`.
+Then, the folks got involved in solving hackathon challenges, from porting applications as `static-pie-apps` to fixing some existing issues).
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:23]:
```suggestion
Then, the folks created teams of 3-4 people, got an entry for each team in [the scoreboard](https://leaderboardhq.com/3wt61mcc) then then started solving [hackathon challenges](https://unikraft.org/community/hackathons/2022-10-munich/challenges/).
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Many more to come!
+
+{{< figure
+    src="/assets/imgs/2022-10-hack-munich-participants.png"
+    title="Participants for the hackathon at Technische Universität München "
+    position="center"
+>}}
+
+
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:38]:
```suggestion
```

**Review by razvand — CHANGES_REQUESTED** [review]
@RaduNichita , see my requests.

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+date = "2022-10-26T03:56:00+08:00"
+author = "Radu Nichita, Razvan Deaconescu"
+tags = ["hackathons", "munich", "unikernel"]
++++
+
+### Grüß Gott, München!
+
+Last weekend, we had yet another successful hackathon with plenty of hard work.
+But this one was special since it was organized in München.
+It was the 3rd international onsite hackathon, after the ones in Lyon and Aachen.
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:12]:
```suggestion
It was the 3rd international onsite hackathon, after the ones in [Lyon](community/hackathons/2022-05-lyon/) and [Aachen](community/hackathons/2022-06-aachen/).
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+From the Unikraft community, [Răzvan](https://github.com/razvand/) and [Radu](https://github.com/RaduNichita/) were there, whereas other community members provided support online, on Discord.
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:13]:
```suggestion
From the Unikraft community, [Răzvan](https://github.com/razvand/) and [Radu](https://github.com/RaduNichita/) were there, whereas other community members provided support online, on Discord (special thanks to [Ștefan](https://github.com/StefanJum).
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+Over `30` participants came to Technische Universität München for the 2-day event about unikernels. This is the highest attendance for an onsite Unikraft Hackathon event!
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:15]:
```suggestion
Over `30` participants came to Technische Universität München for the 2-day event about unikernels.
This is the highest attendance for an onsite Unikraft Hackathon event!
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+### Make open-source contributions great again!
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:17]:
```suggestion
### Make Open Source Contributions Great Again!
```

File: content/en/blog/2022-10-25-unikraft-munich-2022.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+Results speak for themselves:
+
+ - 6 PRs submitted as part of the [`static-pie-apps` repository](https://github.com/unikraft/static-pie-apps/pulls)
+ - PR for [version bump of `newlib`](https://github.com/unikraft/lib-newlib/pull/27)
+ - PR for (version bump of `zlib`](https://github.com/unikraft/lib-zlib/pull/4)
+ - 2 issues in the [core `unikraft` repository](https://github.com/unikraft/unikraft/issues)
+
+ All contributions were awarded points that made it to [the scoreboard](https://leaderboardhq.com/3wt61mcc).
+
+### Special thanks
```
Comment (razvand) [inline content/en/blog/2022-10-25-unikraft-munich-2022.md:34]:
```suggestion
### Special Thanks
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @RaduNichita . See my comments. After these are handled, we are ready to approve and publish the PR.

Make sure you spell checked the contents.

**Comment by RaduNichita** [comment]
> Thanks, @RaduNichita . See my comments. After these are handled, we are ready to approve and publish the PR.
> 
> Make sure you spell checked the contents.

I've used Grammarly for the spell check. Hope everything is OK now.

**Review by razvand — APPROVED** [review]
Thanks, @RaduNichita 

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>
Approved-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Pull request merged**